build.sh

```diff
@@ -1,49 +1,62 @@
-#!/bin/bash
+#! /usr/bin/env bash
 
 set -e # exit if a command fails
 set -o pipefail # Will return the exit status of make if it fails
 
-CFLAGS_COMMON="-O0 -g -DXBOX=1" # Compilation flags for all platforms
-POST_BUILD=""
-
 package_windows() { # Script to prepare the windows exe
-    mkdir -p dist
-    cp i386-softmmu/qemu-system-i386.exe dist/xqemu.exe
-    cp i386-softmmu/qemu-system-i386w.exe dist/xqemuw.exe
-    python2 ./get_deps.py dist/xqemu.exe dist
-    strip dist/xqemu.exe
-    strip dist/xqemuw.exe
+	mkdir -p dist
+	cp i386-softmmu/qemu-system-i386.exe dist/xqemu.exe
+	cp i386-softmmu/qemu-system-i386w.exe dist/xqemuw.exe
+	python2 ./get_deps.py dist/xqemu.exe dist
+	strip dist/xqemu.exe
+	strip dist/xqemuw.exe
 }
 
+postbuild=''
+build_cflags='-O3'
+job_count='4'
+
+if [ "${1:0:2}" = '-j' ]
+then
+	job_count="${1:2}"
+	shift
+fi
```

Comment:
> `configure-xqemu.sh --release && make -j 4`

I like it. A lot.

There is no need for `.sh` though, I said nothing for `build.sh` because there was already a `build` dir and it was an easy way to avoid file name collision, but [commandname extensions considered harmful](https://www.talisman.org/~erlkonig/documents/commandname-extensions-considered-harmful/).